src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.cpp

```diff
@@ -63,6 +76,36 @@ tnsr::AA<DataType, Dim, Frame> inverse_spacetime_metric(
   return inverse_spacetime_metric;
 }
 
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::I<DataType, SpatialDim, Frame> shift(
+    const tnsr::aa<DataType, SpatialDim, Frame>& spacetime_metric,
+    const tnsr::II<DataType, SpatialDim, Frame>&
+        inverse_spatial_metric) noexcept {
+  auto shift = make_with_value<tnsr::I<DataType, SpatialDim, Frame>>(
+      spacetime_metric, 0.);
+  for (size_t i = 0; i < SpatialDim; ++i) {
+    for (size_t j = 0; j < SpatialDim; ++j) {
+      shift.get(i) +=
+          inverse_spatial_metric.get(i, j) * spacetime_metric.get(j + 1, 0);
+    }
+  }
+  return shift;
+}
+
+template <size_t SpatialDim, typename Frame, typename DataType>
+Scalar<DataType> lapse(
+    const tnsr::I<DataType, SpatialDim, Frame>& shift,
+    const tnsr::aa<DataType, SpatialDim, Frame>& spacetime_metric) noexcept {
+  auto lapse = make_with_value<Scalar<DataType>>(shift, 0.);
```

Comment:
Similar comment; can `lapse` be directly initialized to `-get<0,0>(spacetime_metric)` ?